### DISCUSSION TITLE
Contributor call: 19 March 2022

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1982] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @lucibytes on our Discord server.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:

Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 19 March 2022 5pm UTC, which is can be found in your local time zone [here](https://savvytime.com/converter/utc-to-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/Mar-19-2022/5pm)
Or check the bot in Discord! #meetings

### Agenda
- Last call for newsletter contributions!
- To be determined

> This discussion will be updated after the call with notes and to-dos.

**Comment by l3xander** [comment, marked as answer]
### Notes
- The newsletter is coming up! @tangerineshark mentioned they may write about showcase photography tips with a thank-you note to @nataliasayang.
- @/triploidtree from the Discord server has the plan to write a blogpost on how to fix and adjust problems that sometimes come from the bicep measurement (which is used to determine a pattern’s armhole, this doesn’t always give an accurate result). This started a discussion on measurement adjustments in general.
    - @joostdecock explained the current armhole problem: the depth of the armhole is currently based on the bicep measurement, which gets pushed down the larger the measurement is, which negatively impacts the fit. There’s an ongoing issue request to add the waist to armhole measurement.
    - Though, even with the added waist to armhole measurement, adjustments may still be needed. As described by @tangerineshark and @bobgeorgethe3rd, adjustments will always be beneficial, a pattern can never be 100% perfect (even with perfect measurements), so @/triploidtree’s knowledge would be of great help here. @joostdecock added that the patterns serve more as a starting point.
    - @joostdecock explained that designers make their patterns with a specific body in mind - usually their own - and that other bodies (like plus-sized or trans bodies) may not get taken into account. Often designers don’t know how to make their pattern suitable for other body types, it would be interesting to see if we can add these adjustments with the help of experienced people.
    - @/triploidtree explained it’s important to figure out who (what body) a pattern was made for and then figure out the ‘limits’ of that. It’s also helpful to understand why a pattern works the way it does, for example by figuring out why a certain measurement was chosen over another measurement (e.g. waist instead of hip measurement) and how we can improve that for other bodies.
    - @bobgeorgethe3rd added that some of the initial pattern designers aren’t around anymore to update their patterns, which creates “orphan patterns”. This also means the designer isn’t around to explain why they made certain choices for their patterns, so we’ll have to try to figure that out ourselves.
    - @bobgeorgethe3rd added that we could put these pointers on the dev site under “things to consider”, but @joostdecock wants to be careful with putting more expectations on new developers, because that can scare them away.
    - Another main point was that there are a lot of recurring questions in the Discord servers that are related to this. This is not a problem, but it would be good if we had some sort of FAQ in a pattern’s documentation, so we can point people in the right direction. We can also make a Discussion post for questions (where messages don’t get lost in the chats).
    - It more or less comes down to this: adjustments for patterns will always be needed. Solutions to problems will, over time, be added in the code or to the documentation, depending on what’s possible. Also see the To-do’s.
- @bobgeorgethe3rd has been working on the Billy Bucket hat! The main issue right now is that the brim is too floppy, but a fix also shouldn’t make it too stiff. Wire could be an option, which especially works if you’re going for a big sun hat. @tangerineshark mentioned that commercial versions often have a thin batting and circles of stitching, which together enforces the brim.


### To-do’s
All related to the pattern adjustments plan.
- for @joostdecock to look into where on the site we will add this new information
- for the solutions to common questions to be put in a FAQ (discord → documentation)
- the ability to show comments on pattern pages, that go like, “I am [name], my body is like [X], this pattern works for me in [this] and [that] way.”
- over time: to look into the code of patterns (especially orphan patterns) and figure out what’s going on

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
